canonical/ubuntu.com
# WD-12843 - rebrand /aws/pro

## Done

Rebrand  /aws/pro. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14266.demos.haus/aws/pro)
- [figma](https://www.figma.com/design/FoWXkNxxdri9EgSsLdtTrH/24.10-AWS-bubble?node-id=558-3894&node-type=FRAME&t=jWRTqD0hNuqlogdz-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12843](https://warthogs.atlassian.net/browse/WD-12843)

[WD-12843]: https://warthogs.atlassian.net/browse/WD-12843?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
